bibcat/llm/metrics.py

```diff
@@ -352,6 +365,37 @@ def append_human_labels_with_mapped_papertype(
     return human_labels
 
 
+def human_labels_when_no_llm_output(missions, human_data, human_labels, ignored_papertype):
+    """Assign human labels when human classifications exist even with no llm output
+
+    Parameters
+    ----------
+    missions: list[str]
+        list of missions
+    human_data: dict[str, str]
+        dictionary values of item["human"], e.g., {"JWST": "SCIENCE"}
+    human_labels: list[str]
+        True labels by human, a list of papertypes before papertype mapping,
+        For example, ["SCIENCE", "MENTION"]
+    ignored_papertype: str, uppercase
+        config.llms.map_papertypes.ignore.upper(), for instance, "NONSCIENCE"
+
+    Returns
+    -------
+    human_labels: list[str]
+        updated human labels based on the presence of human classifications
+    """
+    if not human_data:  # when no human labels found
+        human_labels.extend([ignored_papertype] * len(missions))
+    else:  # account human labels for metrics even if no llm output
+        for mission in missions:
+            if mission in human_data:
+                human_labels = append_human_labels_with_mapped_papertype(human_data, mission, human_labels)
+            else:
+                human_labels.append(ignored_papertype)
+    return human_labels
```

Comment:
the `else` block here is unnecessary and makes this less clean.  if you return `human_labels` in the `if not human_data` block, you can remove the `else` and de-indent the code.